src/validate.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable quotes */
+
+export const check = (todo, id) => {
+  const findTodo = todo.find((item) => id === item.index);
+  findTodo.completed = !findTodo.completed;
+  localStorage.setItem("todo", JSON.stringify(todo));
+};
```

Comment:
- [ ] Your check function actual reflect both on browser and in the local storage. But the issue is that the check on the browser get disappeared when you refresh the page which is not supposed to be. And when you click on the `clear all completed` button, the task from which the check has been disappeared get deleted. This is not the expected behaviour of the app. Kindly look into that and fix it. 

[todo-behavior.webm](https://user-images.githubusercontent.com/26597432/213689107-867375eb-4c8a-4b76-82e4-34f0886239df.webm)

Reply:
- [ ] @francis3400 great job implementing the changes requested, however, the check and line-through decoration disappears after the page reload although the value gets changed, kindly check on your display method tot ensure it displays the status of the tasks correctly
 https://www.loom.com/share/116425ac8d9948ae88b8fe717f8a0171

- [ ] [optional] I noted somebody needs to click the icon in order to add a new task it would be better to allow even the enter button to be pressed to achieve the same objective.